api-handler/dataAccess.js

```diff
@@ -5,12 +5,16 @@ const cul = require("./getAllCultures");
 const hou = require("./getAllHouses");
 const reg = require("./getAllRegions");
 const tit = require("./getAllTitles");
+const pop = require("./getPopularity");
+const rel = require("./support/deadRelations");
 
 const url_cha = 'https://got-api.bruck.me/api/characters/';
```

Comment:
I hope you make this configurable! The API is not gonna be serviced on bruck's server in the final round!